app/controllers/make_a_demo_payment/confirm_controller.js

```diff
@@ -15,6 +15,8 @@ module.exports = (req, res) => {
   let gatewayAccountId = auth.getCurrentGatewayAccountId(req)
 
   if (protoData) {
+    let paymentAmount = protoData.protoPaymentAmount.replace(new RegExp('\\.', 'g'), '')
```

Comment:
Is this correct? Shouldn't you be multiplying the `protoData.protoPaymentAmount` * 100 ? 
Say if I type `10.000` it still means I expect £10 (1000 pence) not £100 (10000 pence)

Reply:
it is checked in previous controller before sent to this one